src/elements/content-sidebar/additional-tabs/MoreTabLoading.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @flow
+ * @file Preview sidebar more tabs loading component
+ * @author Box
+ */
+
+import * as React from 'react';
+import './AdditionalTabs.scss';
+import IconEllipsis from '../../../icons/general/IconEllipsis';
+
+const MoreTabLoading = () => {
+    return (
+        <div className="bdl-AdditionalTabLoading-item">
```

Comment:
Should the class and component names match?

Reply:
The styles are identical between the two components, would you recommend a more general name that applies to both?